Ryabuhin Artiom/02.git.local/02.GIT.Local.md

```diff
@@ -0,0 +1,69 @@
+	bash
```

Comment:
Wrong markdown, 

Please check documentation for code sections
https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks

and https://guides.github.com/features/mastering-markdown/#examples

Reply:
Done